### PR TITLE
Issue #15218: formatted input files for section 7.3 Where javadoc is used to follow google style guide rules

### PR DESCRIPTION
**Pull request by Zopsss**
#15218

https://checkstyle.org/google_style.html#a7.3

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule73wherejavadocrequired/InputJavadocMethodAndMissingJavadocMethod.java [hunk 1]
```diff
@@ -2,205 +2,149 @@
...
+  }
+
+  String foo6() {
+    return "Fooooooooooooooo" + "oooooooo";
+  }
+
+  // violation below 'Missing a Javadoc comment.'
+  public String foo7() {
+    // making the method
+    // exceed "minLineCount" property limit ( 2 )
```
Comment (rdiachenko) [inline src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule73wherejavadocrequired/InputJavadocMethodAndMissingJavadocMethod.java:46]: Same question as in https://github.com/checkstyle/checkstyle/pull/15306#discussion_r1683393414

Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule73wherejavadocrequired/InputJavadocMethodAndMissingJavadocMethod.java:46]: Answer: https://github.com/checkstyle/checkstyle/pull/15306#discussion_r1683846052

**Review by romani — APPROVED** [review]
ok to merge

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**